build.zig

```diff
@@ -70,13 +70,54 @@ pub fn build(b: *std.Build) !void {
     build_options.addOption(SysgpuBackend, "sysgpu_backend", sysgpu_backend);
     build_options.addOption(Platform, "core_platform", core_platform);
 
+    var examples = [_]Example{
+        .{ .name = "core-custom-entrypoint", .deps = &.{} },
+        .{ .name = "core-triangle", .deps = &.{} },
+        .{ .name = "custom-renderer", .deps = &.{} },
+        .{ .name = "glyphs", .deps = &.{ .assets, .freetype } },
+        // .{ .name = "hardware-check", .deps = &.{ .assets, .zigimg } },
+        .{ .name = "piano", .deps = &.{} },
+        .{ .name = "play-opus", .deps = &.{.assets} },
+        // .{ .name = "sprite", .deps = &.{ .zigimg, .assets } },
+        .{ .name = "text", .deps = &.{.assets} },
+    };
+
+    var sysaudio_tests = [_]SysAudioTest{
+        .{ .name = "record" },
+        .{ .name = "sine" },
+    };
+
+    // Setup Steps
+    const docs_step = b.step("docs", "Generate docs");
+    for (&examples) |*example| {
+        example.run_step = b.step(
+            b.fmt("run-{s}", .{example.name}),
+            b.fmt("Run example: {s}", .{example.name}),
+        );
+    }
+    const test_step = b.step("test", "Test Run: Unit Tests");
+    for (&sysaudio_tests) |*sysaudio_test| {
+        sysaudio_test.run_step = b.step(
+            b.fmt("test-sysaudio-{s}", .{sysaudio_test.name}),
+            b.fmt("Test Run: SysAudio {s}", .{sysaudio_test.name}),
```

Comment:
```suggestion
            b.fmt("Test Run: sysaudio {s}", .{sysaudio_test.name}),
```

subproject names like this are always lowercase